### DISCUSSION TITLE
RDi integration

### DISCUSSION THREAD

**Discussion by zkarj735** [IBM/ibmi-tobi#204] [General]
While integration with Code for i is great, places with an RDi investment will want to leverage it, particularly given its comprehensive outlining, language support, and debugger, much of which is missing from Code for i. Furthermore, adoption of Git requires IFS-based source, which complicates existing compile tools - a problem bob could amply address with this integration.

I note the "obsolete" message on the RDi integration page is now 10 months old. Is there a timeline for this important feature?

**Comment by edmundreinhardt** [comment]
RDi has this requriement on their list.  They first wanted to get a version on a recent Eclipse and this was a big investment for them.  Now that this is out, I hope that the requirement to enable BOB will get back to the top of their list.
